src/main/java/edu/kis/powp/jobs2d/drivers/composite/IDriverComposite.java

```diff
@@ -0,0 +1,17 @@
+package edu.kis.powp.jobs2d.drivers.composite;
+import edu.kis.powp.jobs2d.Job2dDriver;
+import edu.kis.powp.jobs2d.drivers.adapter.LineDriverAdapter;
+
+
+public interface IDriverComposite extends Job2dDriver {
+
+
+    @Override
+    void setPosition(int x, int y);
+    @Override
+    void operateTo(int x, int y);
+    void add(Job2dDriver driver);
+    void add(LineDriverAdapter driver);
```

Comment:
why separate add? 
remove could be useful.

Reply:
Remove added. Separate add was removed.